packages/example-log-extension/src/decorators/log.decorator.ts

```diff
@@ -36,9 +38,11 @@ export function getLogMetadata(
   controllerClass: Constructor<{}>,
   methodName: string,
 ): LevelMetadata {
-  return Reflector.getMetadata(
-    EXAMPLE_LOG_BINDINGS.METADATA,
-    controllerClass.prototype,
-    methodName,
+  return (
+    MetadataInspector.getMethodMetadata<LevelMetadata>(
+      EXAMPLE_LOG_BINDINGS.METADATA,
+      controllerClass.prototype,
+      methodName,
+    ) || {level: LOG_LEVEL.OFF}
```

Comment:
Would it make sense to extend `getMethodMetadata` with an additional argument allowing callers to specify the default value? (It's out of scope of this pull request, just asking.)

It can be useful especially when falsey metadata values are allowed, e.g. a number `0`.